pkg/slayers/scion_spec.gobra

```diff
@@ -415,22 +471,22 @@ pure func IsSupportedPkt(raw []byte) bool {
 }
 
 ghost
-requires acc(sl.AbsSlice_Bytes(ub, 0, len(ub)), _)
-requires CmnHdrLen <= len(ub)
+requires acc(sl.AbsSlice_Bytes(ub, 0, length), _)
+requires CmnHdrLen <= length && length <= len(ub)
 decreases
-pure func GetAddressOffset(ub []byte) int {
-	return unfolding acc(sl.AbsSlice_Bytes(ub, 0, len(ub)), _) in
+pure func GetAddressOffset(ub []byte, length int) int {
```

Comment:
I have added `GetAddressOffsetWithinLength(ub []byte, length int)`. Since there are multiple functions starting with the name Get, it's better to handle this in a separate PR.